<!--
  arbe_phoenix_radar_driver Launch 
-->
<launch>
	<node name="arbe_pointcloud_receiver" pkg="arbe_phoenix_radar_driver" type="arbe_pointcloud_receiver" output="screen" required="true">
	</node>

	####  for GDB launch-prefix="gdb -ex run --args" ####
	<node name="arbe_gui" pkg="arbe_phoenix_radar_driver" type="arbe_gui" output="screen" required="true">
		<param name="AntennaHight" value="0.7" />
		<param name="Mode" value = "4d" />
		<param name="ColoringType" value = "Elevation" />
		<param name="RangeType" value = "Mid" />
		<param name="AtcEnabled" value="0" />
		<param name="Threshold3d" value="25" />
		<param name="Threshold4d" value="25" />
		<param name="DynamicAzimuth" value="10" />
		<param name="DynamicElevation" value="10" />
		<param name="MinDoppler" value="0" />
		<param name="MaxDoppler" value="70" />
		<param name="RecordFileName" value="arbe-rosbag" />
	</node>
</launch>
